README.md

```diff
@@ -95,6 +95,8 @@ After you're finished please remove all the comments and instructions!
 
 - [Live API link](https://greeting-t0aj.onrender.com/greetings)
 
+- [Frontend Reopository](https://github.com/principlebrothers/hello_react)
+
```

Comment:
- [ ] Actually, you created a user from the seed and then useing a look to create a random greeting to the user from the greetings array, ... here is your code:

```ruby
user = User.create(name: 'John Doe')

%w[Hello Hi Howdy Hey Greetings].each do |greeting|
  user.greetings.create(msg: greeting)
end
```

- [ ] However, this is not what was required to do, According to the project's requirement you should have:
1. Create a controller with one action.
- eg:
```ruby
class GreetingsController < ApplicationController
  def index
    render json: Greeting.all....
  end
end
```
2. Create a model.
- eg :
```ruby
class Greeting < ApplicationRecord
end
```
3. Generate a gettings migration file for a table.
- eg: `db/migrate/20221122212005_create_greetings.rb`

### Kindly make sure to implement this.

Reply:
[Ben](@BenMukebo), thanks for the review. There are a few concern

1. My seed file was to help add the greetings to the database but the good news is that you don't need it because I deployed my project and you can use the API.
2. There was no strict rules regarding creating one action for the controller. And my controllers are working fine so I don't know why commented on it to start with.
3. I all the things you pointed out is done so I am confused why you pointed them out. With the exception of the Route Which I didn't add

Reply:
@principlebrothers why do you want me to check all your branches? isn't the "deployment" which is the current branch you submitted that was supposed to contain all your commits?

Reply:
> @principlebrothers why do you want me to check all your branches? isn't the "deployment" which is the current branch you submitted that was supposed to contain all your commits?

I understand your explanation now.

Reply:
Great 👍🏻